Libs/DF/spells.lua

```diff
@@ -1415,8 +1415,8 @@ elseif (DF.IsWotLKWow()) then
 	}
 	DF.FeastIDs = {}
 	DF.RuneIDs = {}
-
-elseif (DF.IsClassicWow()) then
+--~Cata temp
+elseif (DF.IsClassicWow() or DF.IsCataWow()) then

```

Comment:
Marked temporary setup items to deal with objects not existing in cata beta as `~Cata temp` to make for finding them easier.